File: subroutines/all/subroutine nextchar.f
      subroutine nextchar(line,ifc,len)
      character*(*) line
c     Finds the next nonblank in line
      character*1 tab,ctrlM
      common /tab/ tab,ctrlM
      if (ifc .gt. len-1) return
      ifc1=ifc
      do i=ifc1,len-1
        ifc=i
c       if (line(i:i) .eq. ' ') print *,'nextc i=',i,' blank'
c       if (line(i:i) .eq. tab) print *,'nextc i=',i,' tab'
        if (line(i:i) .ne. ' ' .and. line(i:i) .ne. tab) then
c         if (line(i:i) .eq. '!') ifc=len
          return
        end if
      end do
      ifc=len
      return
      end

File: fortran to python conversion/dependencies/subroutine nextchar.py
def nextchar(line, ifc, len):
    """

    Finds the next nonblank in line

    ***
    Parameters:
    line:
    ifc:
    len:

    Returns:
    ifc:

    """
    if (ifc > len - 1):
        return ifc
    ifc1 = ifc
    for i in range(ifc1, len-1):
        ifc = i
        if (line[i:i+1] != " " and line[i:i+1] != "\t"):
            return ifc
    return len
